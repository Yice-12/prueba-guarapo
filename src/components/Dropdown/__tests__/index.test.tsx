import { render } from "@testing-library/react";
import { Dropdown } from "../";

test("render Dropdown", () => {
  const mockOnchange = jest.fn();

  const { asFragment } = render(
    <Dropdown
      options={["test 1"]}
      name="test"
      onChange={mockOnchange}
      placeHolder="test"
    />
  );

  expect(asFragment()).toMatchSnapshot();
});
