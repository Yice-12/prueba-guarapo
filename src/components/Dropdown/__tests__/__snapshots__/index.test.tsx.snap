// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Dropdown 1`] = `
<DocumentFragment>
  <div
    class="dropdown"
  >
    <div
      class="dropdown__content"
    >
      <select
        name="test"
      >
        <option
          hidden=""
          value=""
        >
          test
        </option>
        <option>
          test 1
        </option>
      </select>
      <div
        class="dropdown__img-arrowDropdown"
      >
        <img
          alt="arrowDropdown"
          src="[object Object]"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
